code/nrf-connect/prstlib/include/prstlib/led.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _PRST_LED_H_
+#define _PRST_LED_H_
+
+#include <drivers/gpio.h>
+#include <logging/log.h>
```

Comment:
Including those without a `zephyr/` prefix in the path requires `CONFIG_LEGACY_INCLUDE_PATH` which is deprecated, builds fail against the `main` branch of nrf-connect. It is probably wise to switch to the new style of includes, `$ZEPHYR_BASE/scripts/utils/migrate_includes.py` can help.

Reply:
Excellent catch, thank you! `migrate_includes.py` worked like a charm.

For documentation, [here are](https://docs.zephyrproject.org/latest/releases/release-notes-3.2.html#api-changes) the relevant API changes from Zephyr 3.2.0.